src/components/tab-group/ink-bar.ts

```diff
@@ -0,0 +1,23 @@
+import {Directive, Renderer, ElementRef} from '@angular/core';
+
+@Directive({
+  selector: 'md-ink-bar',
+})
+export class MdInkBar {
+  constructor(public renderer: Renderer, public elementRef: ElementRef) {}
+
+  alignToElement(element: HTMLElement) {
+    this.renderer.setElementStyle(this.elementRef.nativeElement, 'left',
+        this.getLeftPosition(element));
+    this.renderer.setElementStyle(this.elementRef.nativeElement, 'width',
+        this.getElementWidth(element));
+  }
+
+  getLeftPosition(element: HTMLElement): string {
+    return element.offsetLeft + 'px';
```

Comment:
Does `ElementRef` actually have `offsetLeft` and `offsetWidth` properties? I think you actually need to pull them through the renderer.